rosdep/base.yaml

```diff
@@ -4617,6 +4617,11 @@ nkf:
   fedora: [nkf]
   gentoo: [app-i18n/nkf]
   ubuntu: [nkf]
+nlohmann-json:
```

Comment:
Since you are going for the development packages, this key should be `nlohmann-json-dev`.  That way, we can add (either in this PR, or a later one) a separate key of `nlohmann-json` which is just the runtime portions.

Reply:
+1

Reply:
Thanks! I fixed in https://github.com/ros/rosdistro/pull/24432/commits/8178b1406ac6142cf17d622437727cdb1a75fe6f